packages/uploadthing/src/_internal/client-future.ts

```
@@ -1,5 +1,7 @@
-import { Array, Micro, Predicate } from "effect";
+import * as Array from "effect/Array";
```

Biome (1.9.4)

[error] Do not shadow the global "Array" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)